src/libs/actions/IOU.ts

```diff
@@ -8167,6 +8172,124 @@ function mergeDuplicates(params: TransactionMergeParams) {
     API.write(WRITE_COMMANDS.TRANSACTION_MERGE, params, {optimisticData, failureData});
 }
 
+function resolveDuplicates(params: TransactionMergeParams) {
```

Comment:
```suggestion
/** Instead of merging the duplicates, it updates the transaction we want to keep and puts the others on hold without deleting them */
function resolveDuplicates(params: TransactionMergeParams) {
```